DESCRIPTION

```diff
@@ -30,21 +30,22 @@ Authors@R: c(
            role = c("ctb"),
            email = "robertdavey@carpentries.org",
            comment = c(ORCID = "0000-0002-5589-7754")),
-    person(given = "Robert",
```

Comment:
Not sure how I ended up twice, this is a problem in main varnish desc too!

https://github.com/carpentries/varnish/blob/main/DESCRIPTION

Reply:
Ah blame suggests the maintainer added me a second time whilst finalising a release.

Reply:
https://github.com/carpentries/varnish/pull/127

Have PR'd it upstream, possible conflict in future.